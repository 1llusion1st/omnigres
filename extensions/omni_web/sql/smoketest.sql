select omni_web.parse_query_string(null) is null as is_true;
select omni_web.parse_query_string('');
select omni_web.parse_query_string('a');
select omni_web.parse_query_string('a=x&b=1');
select omni_web.parse_query_string('a=x&b=1');
select omni_web.parse_query_string('a&b=1');
select omni_web.parse_query_string('a=%20&b=1');
select omni_web.parse_query_string('a=%20&b=1+3');
