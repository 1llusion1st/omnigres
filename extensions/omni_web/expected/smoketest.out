SELECT omni_web.parse_query_string(NULL) IS NULL AS is_true;
 is_true 
---------
 t
(1 row)

SELECT omni_web.parse_query_string('');
 parse_query_string 
--------------------
 {}
(1 row)

SELECT omni_web.parse_query_string('a');
 parse_query_string 
--------------------
 {a,NULL}
(1 row)

SELECT omni_web.parse_query_string('a=x&b=1');
 parse_query_string 
--------------------
 {a,x,b,1}
(1 row)

SELECT omni_web.parse_query_string('a=x&b=1');
 parse_query_string 
--------------------
 {a,x,b,1}
(1 row)

SELECT omni_web.parse_query_string('a&b=1');
 parse_query_string 
--------------------
 {a,NULL,b,1}
(1 row)

SELECT omni_web.parse_query_string('a=%20&b=1');
 parse_query_string 
--------------------
 {a," ",b,1}
(1 row)

SELECT omni_web.parse_query_string('a=%20&b=1+3');
 parse_query_string 
--------------------
 {a," ",b,"1 3"}
(1 row)

